README.md

```diff
@@ -28,11 +28,11 @@ Kata Containers is working to build a global, diverse and collaborative communit
 
 ## Join Us
 
-You can join our community on any of several places:
+You can join our community on any of the following places:
 
-* [Mailing List](http://lists.katacontainers.io/)
-* IRC on IRC.freenode.net: [#kata-dev](http://webchat.freenode.net/?channels=kata-dev) and [#kata-general](http://webchat.freenode.net/?channels=kata-general)
-* You can get an [invite to our Slack channel](http://bit.ly/KataSlack), and then [join us on Slack](https://katacontainers.slack.com/).
+* Via our [mailing list](http://lists.katacontainers.io/), or
```

Comment:
I would suggest removing the ", or" from the list because that construction goes against the guidelines for creating lists. I would also suggest replacing "Via" with an action verb to keep the list parallel (items 2-4 already fall in line with that suggestion). Then, add a period at the end of each bullet line if you use full sentences for all or skip the period on all if you decide not to use full sentences.

Reply:
How about if you just edit it?  Or is that turned off in this repo?

Reply:
@jberkus: @klynnrif will make edits directly. Thanks.

Reply:
@jberkus Just make sure to add her as a collaborator to your fork for her to be able to push the commit with the edits. Thanks again.

Reply:
Hello @jberkus I have added in edits for you to approve or comment on. Thanks! cc: @rcaballeromx